src/DataBus/SettingsKeys.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace NServiceBus.DataBus.AzureBlobStorage
+{
+    static class SettingsKeys
+    {
+        public const string BlobClient = "Databus.BlobClient";
```

Comment:
It would be good for the key to represent what it actually is:

```suggestion
        public const string BlobContainerClient= "Databus.BlobContainerClient";
```